src/data/panel.ts

```diff
@@ -22,14 +22,19 @@ export const getPanelTitle = (hass: HomeAssistant): string | undefined => {
     return;
   }
 
+  if (panel.url_path === "lovelace") {
+    return hass.localize("panel.states");
+  }
+
   if (panel.url_path === "profile") {
     return hass.localize("panel.profile");
   }
 
-  return (
-    hass.localize(`panel.${panel.title}`) ||
-    panel.title ||
-    // default panel
-    (hass.panels[localStorage.defaultPage] || hass.panels[DEFAULT_PANEL]).title!
```

Comment:
Why is this removed? Isn't it needed anymore? I would think it is still needed when `lovelace` is not the default panel?